internal/command/deploy/deploy.go

```diff
@@ -155,13 +155,26 @@ func DeployWithConfig(ctx context.Context, appConfig *appconfig.Config, args Dep
 	case err != nil:
 		return err
 	case isV2App:
-		if err := appConfig.EnsureV2Config(); err != nil {
+		if err = appConfig.EnsureV2Config(); err != nil {
 			return fmt.Errorf("Can't deploy an invalid v2 app config: %s", err)
 		}
-		return deployToMachines(ctx, appConfig, appCompact, img)
+		err = deployToMachines(ctx, appConfig, appCompact, img)
```

Comment:
same as above

Reply:
Thanks for that--I wasn't sure whether to prefer re-assignment or whether I should just use `:=` everywhere. Fixed.